### PR TITLE
OBJ-230 action codes for objections

### PR DESCRIPTION
**Pull request by mattch1**
Updated status code and added an action codes field and enum to the Objection model.

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/Objection.java [hunk 1]
```diff
@@ -78,6 +85,8 @@ public Objection build() {
     private List<Attachment> attachments = new ArrayList<>();
     @Field("status")
     private ObjectionStatus status;
+    @Field("action_code")
+    private ActionCode actionCode;
```
Comment (markpit) [inline src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/Objection.java:89]: we want to store the actual number in mongo

Comment (mattch1) [reply src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/Objection.java]: Changed field type.

**Review by ICoxCH — APPROVED** [review]
Looks good to me

**Review by mwestacott — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/ActionCode.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.entity;
+
+public enum ActionCode {
```
Comment (markpit) [inline src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/ActionCode.java:3]: we don't need the enum anymore if we are putting the number into the DB

Comment (mattch1) [reply src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/ActionCode.java]: Removed this.

**Review by markpit — APPROVED** [review]

**Pull request merged**